include/yaml-cpp/exceptions.h

```diff
@@ -15,7 +15,7 @@
 
 // This is here for compatibility with older versions of Visual Studio
 // which don't support noexcept
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
There should be changes only related to hunterization. I do expect to see CMake code updates and no C++ code updates, see: https://docs.hunter.sh/en/latest/creating-new/patch.html